packages/wethegit-components/src/components/accordion/component/AccordionItem.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useRef } from "react"
+
+import { Icon, IconDefs, IconSymbol, Tag, Text } from "@local/components"
+
+import styles from "../accordion.module.scss"
+
+const DEFAULT_TAG = "h3" as const
+
+export function GenericSection({ ...props }) {
+  const { as = DEFAULT_TAG, ...rest } = props
+
+  return <Tag as={as} {...rest} />
+}
+
+export interface CardsData {
+  title: string
+  text: string | object
+}
+
+interface Props {
+  handleToggle: (index: number) => void
+  active: number | null
+  card: CardsData
+  icon: string
+  index: number
+}
+
+const AccordionItem: React.FC<Props> = ({ index, active, handleToggle, card, icon }) => {
+  const contentEl = useRef<HTMLDivElement>(null)
+  const { title, text } = card
+
+  if (!card) return null
```

Comment:
You are de-structuring the `card` variable on the line above, so it means it exists. Move that after this check otherwise it will throw.